governance/votes/Executive vote - May 10, 2023.md

```diff
@@ -97,8 +98,10 @@ As per [this poll](https://vote.makerdao.com/polling/QmYFfRuR) the following par
 
 #### [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) Changes
 
-* Increase the WSTETH-A `ttl` by 21,600 seconds from 21,600 seconds to **43,200 seconds**.
-* Increase the WSTETH-B `ttl` by 28,800 seconds from 28,800 seconds to **57,600 seconds**.
+* Increase the WSTETH-A `ttl` by 21,600 seconds (6 hours) from 21,600 seconds (6 hours) to **43,200 seconds** (12 hours).
```

Comment:
I would honestly just cut the seconds altogether and go with only hours, but whatever you prefer.